actors/builtin/verifreg/verified_registry_actor.go

```diff
@@ -109,14 +109,16 @@ func (a Actor) RemoveVerifier(rt vmr.Runtime, verifierAddr *addr.Address) *adt.E
 }
 
 type AddVerifiedClientParams struct {
-	Address   addr.Address
+	Address   addr.Address // PARCHECK: could be a bogus address
```

Comment:
This will be solved along with https://github.com/filecoin-project/specs-actors/issues/556.